frontend/routes/viewer/components/views/components/viewItem/viewItem.component.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ *  Copyright (C) 2017 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import * as React from 'react';
+import { Formik, Field } from 'formik';
+
+import { renderWhenTrue } from '../../../../../../helpers/rendering';
+import { Image } from '../../../../../components/image';
+
+import {
+	ViewpointItem,
+	StyledForm,
+	ThumbnailPlaceholder,
+	IconsGroup,
+	StyledDeleteIcon,
+	StyledEditIcon,
+	StyledCancelIcon,
+	NewViewpointName,
+	Name,
+	NameRow,
+	SaveIconButton,
+	StyledSaveIcon
+} from './viewItem.styles';
+
+interface IProps {
+	viewpoint: any;
+	active: boolean;
+	editMode: boolean;
+	teamspace: string;
+	modelId: string;
+	onCancelEditMode: () => void;
+	onSaveEdit: (values) => void;
+	onDelete?: (event) => void;
+	onOpenEditMode?: () => void;
+	handleClick?: (viewpoint) => void;
```

Comment:
- Can we rename **handleClick** to **onClick** ?

Reply:
Yes, definitely. I don't know why it was named like this. :/